editor/src/components/editor/store/dispatch-strategies.spec.tsx

```diff
@@ -836,66 +838,37 @@ describe('only update metadata on SAVE_DOM_REPORT', () => {
               if (interactionSession == null) {
                 return strategyApplicationResult([])
               }
-              expect(canvasState.startingMetadata).toBe(interactionSession.latestMetadata)
-              expect(canvasState.startingAllElementProps).toBe(
+              expect(canvasState.startingMetadata).not.toBe(interactionSession.latestMetadata)
+              expect(canvasState.startingAllElementProps).not.toBe(
                 interactionSession.latestAllElementProps,
               )
 
+              // first we make sure the _starting_ metadata and startingAllElementProps have the original undefined backgroundColor
+              expect(
+                canvasState.startingMetadata[EP.toString(targetElement)].computedStyle
+                  ?.backgroundColor,
+              ).toBeUndefined()
+              expect(
+                canvasState.startingAllElementProps[EP.toString(targetElement)].style
+                  .backgroundColor,
+              ).toBeUndefined()
+
+              // then we check that the latestMetadata and latestAllElementProps have a backgroundColor defined, as a result of the previous toggleProperty dispatch
+              expect(
+                interactionSession.latestMetadata[EP.toString(targetElement)].computedStyle
+                  ?.backgroundColor,
+              ).toBeDefined()
+              expect(
+                interactionSession.latestAllElementProps[EP.toString(targetElement)].style
+                  .backgroundColor,
+              ).toBeDefined()
               return strategyApplicationResult([])
             },
           },
         ],
       ],
     )
 
-    // toggling the backgroundColor to update the metadata
-    await renderResult.dispatch(
-      [toggleProperty(targetElement, toggleStylePropPaths(toggleBackgroundLayers))],
-      true,
-    )
-
-    // dispatching a no-op change to the interaction session to trigger the strategies
-
-    await renderResult.dispatch([CanvasActions.updateDragInteractionData({})], true, [
-      (canvasState: InteractionCanvasState, interactionSession: InteractionSession | null) => [
-        {
-          id: 'TEST_STRATEGY',
-          name: 'Test Strategy',
-          controlsToRender: [],
-          fitness: 10,
-          apply: function (): StrategyApplicationResult {
-            if (interactionSession == null) {
-              return strategyApplicationResult([])
-            }
-            expect(canvasState.startingMetadata).not.toBe(interactionSession.latestMetadata)
-            expect(canvasState.startingAllElementProps).not.toBe(
-              interactionSession.latestAllElementProps,
-            )
-
-            // first we make sure the _starting_ metadata and startingAllElementProps have the original undefined backgroundColor
-            expect(
```

Comment:
removing these expects defeats the point of this test. please instead create a setup where we already have a passing drag interaction so this TEST_STRATEGY can become active

Reply:
I've just spoken to Balazs about this. I tried doing this and was seeing some unexpected and hard to debug issues around missing `latestMetadata` for the affected element. I think it ultimately stems from this not being a browser test. Either way this needs further investigation. I'm going to add a FIXME into this test and capture this in one of the paper docs so that it's not forgotten.